src/Resources/config/grid/order.yml

```diff
@@ -2,33 +2,85 @@
 # (c) Paweł Jędrzejewski
 
 sylius_grid:
-  templates:
-      action:
-        shop_quick_return: "@SyliusShop/Grid/Action/quickReturn.html.twig"
-  grids:
-    sylius_shop_account_order:
-      actions:
-        item:
-          quick_return:
-            type: shop_quick_return
-            label: bitbag_sylius_inpost_plugin.ui.quick_return
-            options:
-              link:
-                route: sylius_shop_account_order_return
-
-          show:
-            type: shop_show
-            label: sylius.ui.show
-            options:
-              link:
-                route: sylius_shop_account_order_show
-                parameters:
-                  number: resource.number
-          pay:
-            type: shop_pay
-            label: sylius.ui.pay
-            options:
-              link:
-                route: sylius_shop_order_show
-                parameters:
-                  tokenValue: resource.tokenValue
+    templates:
```

Comment:
What's the reason of so many lines changed? I can't read what exactly was the logic change...

Reply:
Ctrl+ALt+L and changed for 4 spaces as tab.
My fault. I will remember for the future.